testing/helpers/jszipMock.js

```diff
@@ -0,0 +1,33 @@
+class Folder {
+    constructor() {
+        this.children = {};
```

Comment:
Maybe need rename "children" to "files"

Reply:
There are 'this._zip.folder(XL_FOLDER_NAME).folder(WORKSHEETS_FOLDER).file(WORKSHEET_FILE_NAME, xmlResult.join(""));' entries and 'folders' can contain 'folders'.